### PR TITLE
EWC-5-DocsUpdate | Api Docs Updated in EWC

### PR DESCRIPTION
**Pull request by sureshCelestial**
Hi Marc,
In this PR we have updated the API Docs of following components:

Panel
RadioGroup
CheckBoxGroup
Dirty
Column
Text
Toggle
Field

Kindly review.

Thanks,
Suresh

### CODE REVIEW

File: api-docs/modern/modern/src/field/Field.js [hunk 1]
```diff
@@ -235,6 +235,28 @@
...
+ /**
+ * @cfg {Ext.data.field.Field/Object/String} dataType
+ * A config for a {@link Ext.data.field.Field} or data field sub-class instance
+ * used to serialize this field's value for form submission. This is used by the
+ * {@link #serialize} method unless `modelValidation` is used, in which case, the
+ * data field of the bound {@link Ext.data.Model model} is used.
+ *
+ * For example:
+ *
+ *      {
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/Field.js]: Should this be in tag format?

File: api-docs/modern/modern/src/field/Text.js [hunk 2]
```diff
@@ -195,3 +210,19 @@
  *
  * @accessor
  */
+
+/**
+ * Called when a key has been pressed in the `<input>`
+ * @protected
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/Text.js]: There is something missing with this jsdoc section, such as the method name, is something missing?

File: api-docs/modern/modern/src/field/Dirty.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
+/**
+ * A mixin that adds `dirty` config and `dirtychange` event to a component (typically a
+ * `field` or `form`).
+ * @private
+ * @since 7.0
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/Dirty.js]: Something is missing in this definition, can you find it?

File: api-docs/modern/modern/src/field/FieldGroupContainer.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
+/**
+ * This Group Container Field is useful for containing multiple form fields
+ * to be a single group and will line up nicely with group other fields.
+ * A common use is for grouping a set of related fields.
+ */
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/FieldGroupContainer.js]: Something is missing from this block definition, can you figure out what it is?

File: api-docs/modern/modern/src/field/RadioGroup.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
+/**
+ * This Field is useful for containing multiple {@link Ext.field.Radio radiofield}.
+ *
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/RadioGroup.js]: Something is missing here too from the block definition?

File: api-docs/modern/modern/src/grid/GridDragZone.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
+/**
+ * @private
```
Comment (branflake2267) [inline api-docs/modern/modern/src/grid/GridDragZone.js]: Is this an orphan?  Something is missing here?

File: api-docs/modern/modern/src/grid/GridDragZone.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
+/**
```
Comment (branflake2267) [inline api-docs/modern/modern/src/grid/GridDragZone.js]: The class block definition is missing, can you add it?

File: api-docs/modern/modern/src/grid/column/Drag.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
+/**
+ * A grid column used by the {@link Ext.grid.plugin.RowDragDrop RowDragDrop} plugin.
```
Comment (branflake2267) [inline api-docs/modern/modern/src/grid/column/Drag.js]: There is something missing here for the class definition block, can you add it?

File: api-docs/packages/froala-editor/src/froala/EditorField.js [hunk 9]
```diff
@@ -0,0 +1,18 @@
+/**
+ * A field version of {@link Ext.froala.Editor}. This allows you to use the Froala Editor
+ * within a form and automatically have its name and value included in a form submit.
```
Comment (branflake2267) [inline api-docs/packages/froala-editor/src/froala/EditorField.js]: There is something missing here for the class definition, can you add it?

File: api-docs/modern/modern/src/field/CheckboxGroup.js [hunk 10]
```diff
@@ -1,35 +1,70 @@
 /**
  * This Field is useful for containing multiple {@link Ext.field.Checkbox checkbox}.
```
Comment (branflake2267) [inline api-docs/modern/modern/src/field/CheckboxGroup.js]: The class definition @decorations seem to be missing, is something for the class definition block?

**Comment by branflake2267** [comment]
Several of the classes are missing class definition decorators, could you add them? 

https://github.com/sencha/ext-web-components/blob/ext-components-7.0.x/api-docs/modern/modern/src/Button.js#L2 - example

**Comment by sureshCelestial** [comment]
@branflake2267 , We have resolved all the changes suggested by you in all files.

Please review.

Thanks

File: api-docs/modern/modern/src/grid/GridDragZone.js (hunk 6 above)
Comment (sureshCelestial) [reply api-docs/modern/modern/src/grid/GridDragZone.js]: Yes, We have added @class and @extend in docblock.

**Pull request merged**